requirements/main.txt

```diff
@@ -3,4 +3,4 @@ lesscpy
 htmlmin
 jsmin
 xxhash
-pyyaml==5.3.1
+pyyaml>=5.4
```

Comment:
```suggestion
pyyaml
```
Let's leave it unpinned, and hopefully `pip` will figure out which version is most suitable.